test/e2e/pkg/testnet.go

```diff
@@ -281,14 +281,12 @@ func NewTestnetFromManifest(manifest Manifest, file string, ifd InfrastructureDa
 	}
 
 	// Set up genesis validators. If not specified explicitly, use all validator nodes.
-	if testnet.Validators == nil || len(*testnet.Validators) == 0 {
-		validatorsMap := make(map[string]int64)
```

Comment:
given that the `validatorsMap[node.Name] = 100` causes an "assignment to nil map" panic, I think we must keep the  nil check + initialization if needed.